jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/codeAction/proposal/quickfix/AddAnnotationMissingQuickFix.java

```diff
@@ -0,0 +1,119 @@
+/*******************************************************************************
+* Copyright (c) 2021 IBM Corporation and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     IBM Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.lsp4jakarta.jdt.codeAction.proposal.quickfix;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.jdt.core.IJavaElement;
+import org.eclipse.jdt.core.dom.ASTNode;
+import org.eclipse.jdt.core.dom.IBinding;
+import org.eclipse.jdt.core.dom.VariableDeclarationFragment;
+import org.eclipse.jdt.internal.corext.dom.Bindings;
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4jakarta.jdt.codeAction.IJavaCodeActionParticipant;
+import org.eclipse.lsp4jakarta.jdt.codeAction.JavaCodeActionContext;
+import org.eclipse.lsp4jakarta.jdt.codeAction.proposal.ChangeCorrectionProposal;
+import org.eclipse.lsp4jakarta.jdt.codeAction.proposal.ModifyAnnotationProposal;
+import org.eclipse.lsp4jakarta.jdt.codeAction.proposal.ModifyModifiersProposal;
+
+/**
+ * Quickfix for adding missing attributes to annotations
+ * 
+ * @author Zijian Pei
+ *
+ */
+public class AddAnnotationMissingQuickFix implements IJavaCodeActionParticipant{
```

Comment:
This class should be renamed to clarify that it adds missing attributes to annotations. Perhaps something like `InsertAnnotationAttributesQuickFix`

Reply:
solved